src/scenechange/mod.rs

```diff
@@ -104,237 +136,289 @@ impl SceneChangeDetector {
       return false;
     }
 
-    if self.exclude_scene_flashes {
-      self.exclude_scene_flashes(frame_set, input_frameno, previous_keyframe);
-    }
-
-    self.is_key_frame(
-      frame_set[0].clone(),
-      frame_set[1].clone(),
-      input_frameno,
-      previous_keyframe,
-    )
-  }
+    // Initially fill score deque with forward frames
+    // ititiallization is different depending on frame set length
```

Comment:
```suggestion
    // Initialization is different depending on frame set length
```
Need to fix the typo+ maybe a rewording is a good idea, like merging both sentences? 
Initially fill score deque with forward frames based on frame_set length

Reply:
Fixed typos